perllib/FixMyStreet/Cobrand/Brent.pm

```diff
@@ -547,6 +542,40 @@ sub waste_munge_request_form_data {
     }
 }
 
+sub available_permissions {
+    my $self = shift;
+
+    my $perms = $self->next::method();
+    $perms->{Waste}->{can_pay_with_csc} = "Can use CSC to pay for subscriptions";
+
+    return $perms;
+}
+
+sub waste_staff_choose_payment_method { 1 }
+sub waste_cheque_payments { 0 }
+
+use constant GARDEN_WASTE_SERVICE_ID => 317;
+use constant GARDEN_WASTE_PAID_COLLECTION_CONTAINER_TYPE => 1;
+sub garden_service_name { 'Garden waste collection service' }
+sub garden_service_id { GARDEN_WASTE_SERVICE_ID }
+sub garden_current_subscription { shift->{c}->stash->{services}{+GARDEN_WASTE_SERVICE_ID} }
+sub get_current_garden_bins { shift->garden_current_subscription->{garden_bins} }
```

Comment:
Can probably think about refactoring these Echo cobrands to do this a bit nicer somehow at some point.